contracts/BlendSwap.sol

```diff
@@ -23,12 +22,14 @@ contract BlendSwap {
         address to;
         uint amount;
         uint releaseTime;
+        bytes32 secretHash;
```

Comment:
Why do you need it _inside_ the struct?

Reply:
Let's remove this out of struct, agree!

@sashasashasasha151